Library/src/Systems/Organizations/OrganizationSystem.cpp

```diff
@@ -309,21 +415,14 @@ void OrganizationSystem::RemoveUserFromOrganization(const csp::common::Optional<
 	}
 	else
 	{
-		const csp::common::Array<csp::common::String>& CurrentOrganizationIds = GetCurrentOrganizationIds();
+		SelectedOrganizationId = GetCurrentOrganizationId();
 
-		if (CurrentOrganizationIds.Size() == 0)
+		if (SelectedOrganizationId.IsEmpty())
 		{
-			CSP_LOG_ERROR_MSG("OrganizationSystem::InviteToOrganization failed: You do not belong to an Organization.");
```

Comment:
Same question about whether we should preserve the error log message.

Reply:
Agreed, see above.